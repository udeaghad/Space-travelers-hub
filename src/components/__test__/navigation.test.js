import React from "react";
import { BrowserRouter } from 'react-router-dom';
import { render } from "@testing-library/react";
import Navbar from "../NavBar";

describe('Jest test' , ()=> {
    it('Match Snapshot' , ()=> {
        const { container } = render(
            <BrowserRouter>
            <Navbar />
            </BrowserRouter>,
        );
        expect(container.firstChild).toMatchSnapshot();

    })
})