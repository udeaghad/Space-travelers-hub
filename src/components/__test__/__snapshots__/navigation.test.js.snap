// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jest test Match Snapshot 1`] = `
<nav>
  <div
    class="website-header"
  >
    <img
      alt="Logo"
      id="logo"
      src="logo.png"
    />
    <h1>
      Space Travelers Hub
    </h1>
  </div>
  <ul
    class="navItems"
  >
    <li
      class="rocket-link"
    >
      <a
        aria-current="page"
        class="active"
        href="/"
      >
        Rockets
          |
      </a>
    </li>
    <li
      class="mission-link"
    >
      <a
        class=""
        href="/missions"
      >
        Missions
          |
      </a>
    </li>
    <li
      class="profile-link"
    >
      <a
        class=""
        href="/profile"
      >
        Profile
          |
      </a>
    </li>
  </ul>
</nav>
`;
